src/clients/api/queries/useUserMarketInfo.ts

```diff
@@ -142,6 +147,9 @@ const useUserMarketInfo = ({
         collateral,
         percentOfLimit,
         hypotheticalLiquidity: ['0', '0', '0'] as [string, string, string],
+        xvsPerDay: new BigNumber(market.supplierDailyVenus)
+          .plus(new BigNumber(market.borrowerDailyVenus))
+          .div(1e18),
```

Comment:
Just so we don't have constants in our code, let's get the XVS token and use its decimals instead:
```
.div(new BigNumber(10).pow(getToken('xvs').decimals))
```